guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -381,6 +381,85 @@ Before you begin, ensure that the following conditions are met:
 * The NIC is capable of PXE booting. For more information, see http://ipxe.org/appnote/hardware_drivers
 * The NIC is compatible with iPXE
 
+==== Example WIP of Lzap's Blogpost
+
+Using {Project} you can set up and use iPXE if you have the following restrictions that prevent you from using PXE:
+
+* On networks with unmanaged DHCP servers,
+* If the PXE service is blacklisted on your network or restricted by a firewall
+* When the TFTP UDP-based protocol is not reliable, for example, on low-bandwidth networks
+
+With {Project}, you can set up PXE over TFTP to download the `ipxe.efi` file. When the file downloads, all communication continues using HTTP. {Project} uses the iPXE provisioning script either to load an operating system installer or the next entry in the boot order.
+
+When an EFI host boots from PXE, the `ipxe.efi` file changes the process to iPXE. BIOS systems use the `undionly.kpxe` file.
```

Comment:
"to change a process" is weird, a host downloads and executes the code which is the iPXE. So iPXE takes over the process of downloading a iPXE configuration from Foreman using HTTP and then either loads installer or boots from local drive.

Reply:
Removing this sentence because I think that it is already explained in the previous paragraph.